docs/public/presets/toml/jetpack.toml

```diff
@@ -1,14 +1,20 @@
-format = """
-$cmd_duration\
+# version: 1.0.0
+
+add_newline = true
+scan_timeout = 500
+command_timeout = 1000
```

Comment:
This should be set by users themselves rather than by any preset.

```suggestion
```

Reply:
agreed